dotcom-rendering/src/components/AdSlot.apps.tsx

```diff
@@ -99,8 +98,6 @@ const SupportBanner = ({
  * These slots are dynamically inserted into articles using React portals.
  * The ref is important so that we can provide the location of the slot to
  * the native layer, for it to "paint" an advert over the top of it.
- *
- * @todo Style for dark mode
```

Comment:
Nit and non-blocking: Are there tickets for these as well as comments? Todos can get a bit lost after the pr is merged